go.mod

```diff
@@ -86,13 +88,10 @@ require (
 )
 
 replace (
-	github.com/golang/protobuf => github.com/golang/protobuf v1.4.2
```

Comment:
We really tried to upgrade protobuf and grpc (started from argo-rollouts). Here is description of why it is not done yet: https://github.com/argoproj/argo-rollouts/issues/1091